paradise_papers_search/fetch_api/models/Officer.py

```diff
@@ -38,6 +38,23 @@ def serialize_connections(self):
                 'nodes_type': 'entity',
                 'nodes_related': helpers.serialize_relationship(self.entities.all(), 'OFFICER_OF'),
             },
+            {
+                'nodes_type': 'officer',
+                'nodes_related': self._officer_relationships(),
+            },
         ]
 
+    def _officer_relationships(self):
```

Comment:
This looks like the `entity_relationship` function => https://github.com/neo4j-examples/paradise-papers-django/blob/refactor_app/paradise_papers_search/fetch_api/models/Entity.py#L72
Can we add this to the helper file?